dynalab/app/domain/model.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+class ModelController:
+    def __init__(self) -> None:
+        self.initialized = True
```

Comment:
could we have a more realistic example ? Maybe a sample for the NLI task ?

e.g. I want to know how to load model and other resources.

Reply:
excellent idea, let's use the NLI one and create a readme-tutorial this week